src/ol/mapbrowserevent.js

```diff
@@ -30,9 +30,11 @@ goog.require('ol.pointer.PointerEventHandler');
  * @param {string} type Event type.
  * @param {ol.Map} map Map.
  * @param {goog.events.BrowserEvent} browserEvent Browser event.
+ * @param {boolean} dragged Is the map currently being dragged?
  * @param {?olx.FrameState=} opt_frameState Frame state.
  */
-ol.MapBrowserEvent = function(type, map, browserEvent, opt_frameState) {
+ol.MapBrowserEvent = function(type, map, browserEvent, dragged,
```

Comment:
What about changing `dragged` to an optional (`false` by default) parameter?
The `dragged` parameter should only be passed for `ol.MapBrowserEvent.EventType.POINTERDRAG` events: the value is false (by definition) for all the other events.